src/main/java/com/serenitydojo/calculator/Calculator.java

```diff
@@ -1,7 +1,48 @@
 package com.serenitydojo.calculator;
 
+
+
 public class Calculator {
-    public int evaluate(String expression) {
-        return 0;
+    public static int evaluate(String str) throws Exception {
+        str = str.replace(" ", "");
+        if (str.isEmpty()) {
```

Comment:
A simpler approach would be to split the string into space-separated words using the Splitter class

(see for example https://github.com/serenity-dojo/string-calculator-challenge/blob/27368a852d6196332b1aabe3bbc14af885930a80/src/main/java/com/serenitydojo/calculator/Calculator.java#L11)